### PR TITLE
RA-1955:Updating Reference Application Concepts

### PR DESCRIPTION
**Pull request by HerbertYiga**
Updating Reference Application Concepts

Ticket Id
https://issues.openmrs.org/browse/RA-1955

### CODE REVIEW

**Comment by HerbertYiga** [comment]
cc @dkayiwa  @sherrif10

**Comment by dkayiwa** [comment]
@HerbertYiga did you look at the above build failures?

File: api/src/main/resources/Reference_Application_Numeric_Concepts-26-2.x.xml [hunk 1]
```diff
@@ -8,6 +8,6 @@
 <concept_numeric concept_id="5089" hi_absolute="250.0" low_absolute="0.0" units="kg" allow_decimal="true"/>
 <concept_numeric concept_id="5090" hi_absolute="272.0" low_absolute="10.0" units="cm" allow_decimal="true"/>
 <concept_numeric concept_id="5092" hi_absolute="100.0" low_absolute="0.0" units="%" allow_decimal="false"/>
-<concept_numeric concept_id="5242" hi_absolute="999.0" hi_critical="120.0" hi_normal="70.0" low_absolute="0.0" units="" allow_decimal="false"/>
```
Comment (sherrif10) [inline api/src/main/resources/Reference_Application_Numeric_Concepts-26-2.x.xml:11]: Hello @HerbertYiga , Were you not supposed to add a new xml for Reference_Application_Numeric_Concepts-27-2.xml.

Comment (HerbertYiga) [reply api/src/main/resources/Reference_Application_Numeric_Concepts-26-2.x.xml:11]: @sherrif10  should i leave whatever is in Reference_Application_Numeric_Concepts-26-2.x.xml as it is?

**Review by sherrif10 — COMMENTED** [review]
Hey @HerbertYiga  , can we checkout the build failure

**Comment by HerbertYiga** [comment]
hi @dkayiwa  does this look good enough to be merged in?

**Comment by HerbertYiga** [comment]
could the commit message be clear enough here? @dkayiwa

**Comment by sherrif10** [comment]
Hello Herbert. i thought you need to update referencemetadata activator ,DataImporter XML files to be update with current file. [Check and example here for guidance](https://github.com/openmrs/openmrs-module-referencemetadata/blob/master/api/src/main/java/org/openmrs/module/referencemetadata/ReferenceMetadataActivator.java#L102) with in same file

**Comment by tendomart** [comment]
@HerbertYiga  is it me who is not seeing the link to the ticket ?

File: api/src/main/resources/Reference_Application_Numeric_Concepts-26-2.x.xml (hunk 1 above)
Comment (dkayiwa) [reply api/src/main/resources/Reference_Application_Numeric_Concepts-26-2.x.xml:11]: @HerbertYiga what do the wiki page instructions say?

Comment (HerbertYiga) [reply api/src/main/resources/Reference_Application_Numeric_Concepts-26-2.x.xml:11]: @dkayiwa  it says , copy the contents of the generated files to their respective xml files under https://github.com/openmrs/openmrs-module-referencemetadata/blob/master/api/src/main/resources  and its what i did for this pr

**Pull request merged**